tests/benchmarks/test_join.py

```diff
@@ -0,0 +1,57 @@
+import dask.dataframe as dd
+import pytest
+
+from ..utils_test import cluster_memory, timeseries_of_size
+
+mem_mult = [
+    0.1,
+    pytest.param(
+        1,
+        marks=pytest.mark.skip(reason="Does not finish"),
+    ),
+    pytest.param(
+        10,
+        marks=pytest.mark.skip(reason="Does not finish"),
+    ),
+]
+
+
+@pytest.mark.parametrize("mem_mult", mem_mult)  # [0.1, 1, 10]
+def test_join_big(small_client, mem_mult):
+    memory = cluster_memory(small_client)  # 76.66 GiB
+
+    df1_big = timeseries_of_size(
+        memory * mem_mult,
+    )
+    df1_big["x2"] = df1_big["x"] * 1e9
+    df1_big = df1_big.astype({"x2": "int"})
```

Comment:
This is a subtle enough point that we should add a small comment about what this is doing / why it's needed

Reply:
Done : )  see [076ddff](https://github.com/coiled/coiled-runtime/pull/382/commits/076ddff3bbad37e34510d7f1de538ab1cdcee9a2)